.nvmrc

```diff
@@ -1 +1 @@
-11.14.0
+14.15.4
```

Comment:
@gregmercer didn't you want this to be Node version 16?

You don't have to specify the minor or patch number either if you just want to require `16` that would also be ok.

Reply:
@sherakama 
I've been able to update this to 15.14.0. Using npm I haven't been able to go up to 16 without getting this error:
error: no template named 'remove_cv_t' in namespace 'std'; did you mean 'remove_cv'?
Using yarn was the only way I could seem to be on node 16 and get things to work.
I've run across a few folks saying basically this:
It seems to be caused by an incompatibility [between node-sass and Node 16] https://stackoverflow.com/questions/67241196/error-no-template-named-remove-cv-t-in-namespace-std-did-you-mean-remove

Reply:
Hmm. Yeah, It looks like we will have to go to Next 12 to get node-sass support at ^6.0.0 

That sounds like a much bigger refactor/upgrade than we need. Node 14 would probably make more sense as the odd valued releases get removed from support after 6 months and the even valued releases get put into LTS

https://nodejs.org/en/about/releases/